code/game/gamemodes/cult/runes.dm

```diff
@@ -1002,6 +1049,7 @@ var/list/teleport_runes = list()
 		for(var/obj/I in new_human)
 			new_human.unEquip(I)
 		summoned_guys -= new_human
+		ghosts --
```

Comment:
if you do `ghosts++` at least do `ghost--` without the space to keep it consistent...